lib/components/textEditor/richText.js

```diff
@@ -126,23 +125,10 @@ class RichText extends Component {
                     <BoldButton {...externalProps} />
```

Comment:
Should this instead allow buttons to be decided by props? If this component is reused elsewhere (it is used in Secure Reader) then they may not want these updates.

Reply:
I asked a question about this to Jared Crane. He replied that this element should look the same for SecureShare and Secure Reader. I think it makes no sense to add this functionality until it is needed.